charts/app/templates/lpa.yaml

```diff
@@ -17,4 +21,4 @@ kind: ConfigMap
7\x2E\x35\x37\x31\x30\x32\x38\x36\x36\x36\x0A\x32\x38\x30\x2C\x45\x36\x30\x30\x30\x30\x32\x38\x34\x2C\x43\x72\x61\x77\x6C\x65\x79\x20\x4C\x50\x41\x2C\x35\x32\x36\x33\x39\x30\x2C\x31\x33\x37\x35\x38\x31\x2C\x2D\x30\x2E\x31\x39\x35\x33\x33\x2C\x35\x31\x2E\x31\x32\x33\x35\x37\x2C\x34\x34\x39\x37\x31\x32\x30\x34\x2E\x39\x37\x32\x36\x39\x34\x34\x2C\x33\x38\x37\x39\x39\x2E\x37\x32\x33\x32\x37\x39\x35\x37\x34\x35\x0A\x32\x38\x31\x2C\x45\x36\x30\x30\x30\x30\x32\x38\x35\x2C\x48\x6F\x72\x73\x68\x61\x6D\x20\x4C\x50\x41\x2C\x35\x31\x34\x37\x33\x39\x2C\x31\x32\x35\x31\x33\x38\x2C\x2D\x30\x2E\x33\x36\x35\x36\x38\x2C\x35\x31\x2E\x30\x31\x34\x31\x38\x2C\x34\x33\x36\x33\x38\x33\x38\x34\x38\x2E\x30\x38\x35\x39\x30\x37\x2C\x31\x36\x36\x39\x30\x30\x2E\x30\x31\x38\x35\x33\x39\x33\x30\x37\x0A\x32\x38\x32\x2C\x45\x36\x30\x30\x30\x30\x32\x38\x36\x2C\x4D\x69\x64\x20\x53\x75\x73\x73\x65\x78\x20\x4C\x50\x41\x2C\x35\x33\x33\x30\x35\x34\x2C\x31\x33\x30\x36\x32\x33\x2C\x2D\x30\x2E\x31\x30\x32\x37\x32\x2C\x35\x31\x2E\x30\x35\x39\x35\x33\x2C\x32\x39\x36\x36\x36\x33\x36\x38\x31\x2E\x30\x30\x35\x32\x37\x32\x2C\x31\x32\x30\x33\x36\x36\x2E\x36\x39\x32\x31\x34\x33\x30\x31\x38\x0A\x32\x38\x33\x2C\x45\x36\x30\x30\x30\x30\x32\x38\x37\x2C\x57\x6F\x72\x74\x68\x69\x6E\x67\x20\x4C\x50\x41\x2C\x35\x31\x33\x31\x32\x30\x2C\x31\x30\x34\x36\x34\x30\x2C\x2D\x30\x2E\x33\x39\x35\x31\x2C\x35\x30\x2E\x38\x33\x30\x32\x34\x2C\x32\x34\x34\x30\x30\x35\x30\x37\x2E\x32\x36\x36\x30\x38\x36\x36\x2C\x33\x30\x32\x39\x33\x2E\x30\x32\x31\x39\x31\x31\x39\x39\x38\x36\x0A\x32\x38\x34\x2C\x45\x36\x30\x30\x30\x30\x32\x38\x38\x2C\x42\x61\x74\x68\x20\x61\x6E\x64\x20\x4E\x6F\x72\x74\x68\x20\x45\x61\x73\x74\x20\x53\x6F\x6D\x65\x72\x73\x65\x74\x20\x4C\x50\x41\x2C\x33\x36\x36\x32\x31\x37\x2C\x31\x36\x31\x39\x39\x39\x2C\x2D\x32\x2E\x34\x38\x36\x35\x34\x2C\x35\x31\x2E\x33\x35\x36\x30\x34\x2C\x33\x35\x31\x31\x32\x33\x32\x31\x33\x2E\x37\x35\x36\x30\x30\x31\x2C\x31\x34\x31\x36\x38\x34\x2E\x38\x31\x34\x32\x32\x33\x38\x30\x35\x0A\x32\x38\x35\x2C\x45\x36\x30\x30\x30\x30\x32\x38\x39\x2C\x22\x42\x6F\x75\x72\x6E\x65\x6D\x6F\x75\x74\x68\x2C\x20\x43\x68\x72\x69\x73\x74\x63\x68\x75\x72\x63\x68\x20\x61\x6E\x64\x20\x50\x6F\x6F\x6C\x65\x20\x4C\x50\x41\x22\x2C\x34\x31\x30\x38\x31\x35\x2C\x39\x34\x30\x36\x37\x2C\x2D\x31\x2E\x38\x34\x38\x30\x37\x2C\x35\x30\x2E\x37\x34\x36\x30\x39\x2C\x31\x36\x31\x33\x31\x39\x37\x39\x30\x2E\x36\x39\x31\x35\x32\x35\x2C\x31\x34\x36\x39\x32\x38\x2E\x37\x31\x39\x35\x32\x31\x30\x32\x32\x0A\x32\x38\x36\x2C\x45\x36\x30\x30\x30\x30\x32\x39\x30\x2C\x22\x42\x72\x69\x73\x74\x6F\x6C\x2C\x20\x43\x69\x74\x79\x20\x6F\x66\x20\x4C\x50\x41\x22\x2C\x33\x35\x39\x39\x39\x30\x2C\x31\x37\x34\x38\x34\x36\x2C\x2D\x32\x2E\x35\x37\x37\x34\x32\x2C\x35\x31\x2E\x34\x37\x31\x31\x35\x2C\x31\x30\x39\x37\x30\x30\x31\x32\x38\x2E\x36\x39\x34\x37\x35\x32\x2C\x31\x30\x39\x37\x32\x32\x2E\x38\x34\x39\x32\x38\x32\x33\x35\x37\x0A\x32\x38\x37\x2C\x45\x36\x30\x30\x30\x30\x32\x39\x31\x2C\x43\x6F\x72\x6E\x77\x61\x6C\x6C\x20\x4C\x50\x41\x2C\x32\x31\x32\x3
```

Comment:
This is an encoded representation of the following:

All contents of:

https://raw.githubusercontent.com/Planning-Inspectorate/appeal-planning-decision/master/packages/appeals-service-api/data/lpa-list.csv

With a single new line:

```
398,E69999999,System Test Borough Council,408395,438626,-1.87389,53.84382,366419288.83316,135925.597604556
```

The raw text CSV contents were then passed through:

https://onlineunicodetools.com/escape-unicode

I believe the original method was done using python - alas, I am no pythonista. 

**I have manually added the BOM to the start of this output**

Reply:
The missing `\uFEFF` at the start of this string is the byte order mark. Whether I select the option or not, it does not get added. I did not add this in manually as I do not understand the implications of doing so.